plan/config.go

```diff
@@ -104,10 +120,35 @@ func parseBehavior(d string) Behavior {
 		ClientAxis: clientAxis,
 		ParamsAxes: values,
 	}
-
 	return behavior
 }
 
+func parseSkipBehavior(d string) (string, []Filter, error) {
+	pair := strings.SplitN(d, "=", 2)
+	if len(pair) != 2 {
+		return "", nil, fmt.Errorf("Invalid input: %s", d)
```

Comment:
Error messages should start with lower case.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings